docs/server-workflow-search.md

```diff
@@ -109,184 +110,63 @@ There are some pre-allowlisted search attributes that are handy for testing:
 
 ```bash
-tctl --ns samples-namespace workflow start --tq helloWorldGroup --wt main.Workflow --et 60 --dt 10 -i '"vancexu"' -search_attr_key 'CustomIntField | CustomKeywordField | CustomStringField |  CustomBoolField | CustomDatetimeField' -search_attr_value '5 | keyword1 | vancexu test | true | 2019-06-07T16:16:36-08:00'
+docker system prune -a --volumes
 ```
 
-#### Search Workflows with List API
+5. Register a local Namespace and start using it.
 
-```bash
-tctl --ns samples-namespace wf list -q '(CustomKeywordField = "keyword1" and CustomIntField >= 5) or CustomKeywordField = "keyword2"' -psa
 ```
-
-```bash
-tctl --ns samples-namespace wf list -q 'CustomKeywordField in ("keyword2", "keyword1") and CustomIntField >= 5 and CloseTime between "2018-06-07T16:16:36-08:00" and "2019-06-07T16:46:34-08:00" order by CustomDatetimeField desc' -psa
+tctl --ns samples-namespace n re
 ```
 
-To list only open Workflows, add `CloseTime = missing` to the end of the query.
-
-Note that queries can support more than one type of filter:
+6. Allow list search attributes.
 
 ```bash
-tctl --ns samples-namespace wf list -q 'WorkflowType = "main.Workflow" and (WorkflowId = "1645a588-4772-4dab-b276-5f9db108b3a8" or RunId = "be66519b-5f09-40cd-b2e8-20e4106244dc")'
+tctl --ns namespace adm cl asa --search_attr_key NewKey --search_attr_type 1
 ```
 
-```bash
-tctl --ns samples-namespace wf list -q 'WorkflowType = "main.Workflow" StartTime > "2019-06-07T16:46:34-08:00" and CloseTime = missing'
-```
-
-### Web UI Support
-
-Queries are supported in [Temporal Web](https://github.com/temporalio/temporal-web) as of release 0.20.0. Use the "Basic/Advanced" button to switch to "Advanced" mode and type the query in the search box.
-
-## Local Testing
-
-1. Increase Docker memory to higher than 6GB. Navigate to Docker -> Preferences -> Advanced -> Memory
-2. Get the Temporal Docker compose file. Run `curl -L https://github.com/temporalio/temporal/releases/latest/download/docker.tar.gz | tar -xz --strip-components 1 docker/docker-compose-es.yml`
-3. Start Temporal Docker (which contains Apache Kafka, Apache Zookeeper, and Elasticsearch) using `docker-compose -f docker-compose-es.yml up`
-4. From the Docker output log, make sure Elasticsearch and Temporal started correctly. If you encounter an insufficient disk space error, try `docker system prune -a --volumes`
-5. Register a local namespace and start using it. `tctl --ns samples-namespace n re`
-6. Allow list search attributes. `tctl --ns namespace adm cl asa --search_attr_key NewKey --search_attr_type 1`
-
-Note: starting a Workflow with search attributes but without Elasticsearch will succeed as normal, but will not be searchable and will not be shown in list results.
+Note that starting a Workflow with search attributes, but without Elasticsearch, will succeed as normal.
+However it will not be searchable and will not be shown in list results.
```

Comment:
Minus the default attributes right?

Reply:
@alexshtin ?

Reply:
So if we are talking about search attributes, no. None of them are supported w/o ES. If just about general search queries then search by WorkflowId, RunId, WorkflowType, ExecutionStatus, ExecutionTime, CloseTime are always supported.

Reply:
I updated the phrasing here.